MREUnityRuntime/MREUnityRuntimeLib/Patching/Types/PhysicsBridgePatch.cs

```diff
@@ -92,10 +100,104 @@ internal Snapshot ToSnapshot()
 
 		public int TransformCount { get; set; }
 
+		public Snapshot.SnapshotFlags Flags { get; set; }
+
 		/// <summary>
 		/// Serialized as a string in Json.
 		/// https://www.newtonsoft.com/json/help/html/SerializationGuide.htm
 		/// </summary>
 		public byte[] TransformsBLOB { get; set; }
 	}
+
+	public class PhysicsTranformServerUploadPatch : IPatchable
+	{
+		public struct OneActorUpdate
+		{
+			public OneActorUpdate(Guid id,UnityEngine.Vector3 localPos, UnityEngine.Quaternion localQuat,
+				UnityEngine.Vector3 appPos, UnityEngine.Quaternion appQuat)
+			{
+				localTransform = new TransformPatch();
+				appTransform = new TransformPatch();
+
+				localTransform.Position = new Vector3Patch(localPos);
+				localTransform.Rotation = new QuaternionPatch(localQuat);
+
+				appTransform.Position = new Vector3Patch(appPos);
+				appTransform.Rotation = new QuaternionPatch(appQuat);
+				
+				actorGuid = id;
+			}
+
+			public OneActorUpdate(OneActorUpdate copyIn)
+			{
+				localTransform = copyIn.localTransform;
+				appTransform = copyIn.appTransform;
+				actorGuid = copyIn.actorGuid;
+			}
+
+			/// tests if 2 transforms equal
```

Comment:
>/// tests if 2 transforms equal [](start = 3, length = 31)

Would it be useful to override the `Equals` method of the TransformPatch object so that we have the equality testing for the patch in a central location?

Reply:
let's do this for the next PR

---
In reply to: [450379741](https://github.com/microsoft/mixed-reality-extension-unity/pull/210#discussion_r450379741) [](ancestors = 450379741)

Reply:
Sounds good

---
In reply to: [450995094](https://github.com/microsoft/mixed-reality-extension-unity/pull/210#discussion_r450995094) [](ancestors = 450995094,450379741)